### ISSUE TITLE
Remove dependency from Yahoo Finance

### ISSUE THREAD

**Issue by Kriattiffer** [gbeced/pyalgotrade#98]
Yahoo has totally discontinued finance API, so `yahoofinance` module doesn't work anymore.
Luckily, Google Finance is still alive.

**Comment by femtotrader** [comment]
Related issue https://github.com/pydata/pandas-datareader/issues/315
Some solutions for "Python"ers

**Comment by gbeced** [comment]
Yahoo Finance API is broken for a while and I will remove support for it, but I'd also like to re-write the tutorial using a different source.
Google Finance doesn't seem to have adjusted prices.
Any idea of what would be a good data source for historical prices that includes adjusted values ?
Thanks

**Comment by silver0a1** [comment]
Anyone have experience using quandl?

**Comment by femtotrader** [comment]
This is now fixed in latest [pandas-datareader](https://github.com/pydata/pandas-datareader/) (0.5)

```python
import pandas_datareader as pdr
pdr.DataReader("GOOG", "yahoo")  # returns a DataFrame
pdr.DataReader(["IBM", "GOOG"], "yahoo")  # returns a Panel for now but should return a MultiIndexed DataFrame https://github.com/pydata/pandas-datareader/issues/297
```

**Comment by vvug** [comment]
@femtotrader Thanks for the suggestion, but the data from pandas-datareader differs from the data of the tutorial. Check for example the `Adj Close` column of the [tutorial](https://gbeced.github.io/pyalgotrade/docs/v0.18/html/tutorial.html):

```
Date,Open,High,Low,Close,Volume,Adj Close
2000-01-03,124.62,125.19,111.62,118.12,98122000,28.81
```

and the one coming from pandas-datareader (for the same date):

```
Date,Open,High,Low,Close,Adj Close,Volume
2000-01-03,31.15625,31.296875,27.90625,29.53125,26.795395,98114800
```

How is it?

**Comment by pandalibin** [comment]
@gbeced Thanks, so according to your plan, when the updated tutorial replacing the yahoo finance can be online?

**Comment by jakehoskins** [comment]
I have a PR open for making the tutorials work: 

https://github.com/gbeced/pyalgotrade/pull/113

Can update the docs too if you like.

**Comment by gbeced** [comment]
This was implemented in 20d7d963a10b579b966cd716d490379289321368
